proc_stat_io.go

```diff
@@ -0,0 +1,58 @@
+package procfs
+
+import (
+	"fmt"
+	"io/ioutil"
+)
+
+type ProcIO struct {
```

Comment:
Exported fields will need a docstring. `golint` (`go get -u github.com/golang/lint/golint`) will catch this.